share/completions/pnpm.fish

```diff
@@ -0,0 +1,20 @@
+function _pnpm_completion
+  set cmd (commandline -o)
```

Comment:
All of these variables should explicitly be scoped as *local* variables, or you would override any global or universal variable by that name:

```fish
set -l cmd (commandline -o)
```